### ISSUE TITLE
Jackett API key changes

### ISSUE THREAD

**Issue by ngosang** [liaralabs/swizzin#259]
Hello,
We are Jackett developers coming from here => https://github.com/Jackett/Jackett/issues/6858

TLDR; the user said each time Jackett updates / restarts the API key changes and he has to re-configure Sonarr.

The API key is created the first time you run Jackett and it's saved in `/home/YOUR_USER/.config/Jackett/ServerConfig.json`. Jackett never changes it again so, we are supposing the config folder changes in each execution or something like that.

I notice little differences between your script and ours. I'm guessing the user changes with each execution because %I means "instance name" => https://github.com/liaralabs/swizzin/blob/6a872fc8e3a2676ab02fd96c32907feb8223e7a3/scripts/install/jackett.sh#L51
Jackett will go to find the file in this path `/home/%I/.config/Jackett/ServerConfig.json`

The configuration path should be always the same to not lose the config. There are 2 solutions:
* Make the user fixed
* Add this environment var to systemd unit to make the config path fixed => https://github.com/Jackett/Jackett#home-directory

Could you help us? Thank you!

**Comment by liaralabs** [comment]
Hi ngosang, thanks for opening the ticket here.

The Jackett service is actually intended to be started with the username as the instance name, thus my jackett service would be called `jackett@liara`. The `%I` expands to start the service as my user, in my home directory. Thus, the config file gets found in `~/.config/Jackett` and never moves.

I believe this problem is actually more simple than that. I actually tracked this down to a permissions issue in the how the ServerConfig.json is created. Recently I changed the method I use for initializing Jackett. I used to let Jackett create the ServerConfig, but now I cat the framework of the json in and let Jackett fill in the rest when it starts for the first time.

Trouble is, Jackett wasn't able write that file because it was owned by root (the setting of permissions was done in the wrong order), so it could never fill in the details for ServerConfig, thus keeping them in memory until it quit and then generating new ones.

I've ordered the permission setting properly and it looks like ownership is correct now.

I believe this should be fixed now.

**Comment by Cailerh** [comment]
Ran your installer again and rebooted VPS twice and it kept same API key so thank you for helping with this fix :D
